src/Sylius/Bundle/ResourceBundle/Form/DataTransformer/CollectionToStringTransformer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Form\DataTransformer;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\Form\DataTransformerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class CollectionToStringTransformer implements DataTransformerInterface
```

Comment:
IMO we should build `CollectionToArrayTransformer` and `ArrayToStringTransformer` and then we can use composition with `Symfony\Component\Form\Extension\Core\DataTransformer\DataTransformerChain` to chain them into the desired one. This will allow us for greater flexibility.